src/PokemonTrainer.java

```diff
@@ -0,0 +1,39 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class PokemonTrainer {
+    private final String name;
+
+    private static final FirePokemon charizard = new FirePokemon("Charrizard", 76, 150, "firenougats", "GRRRRRRRRRRRRRRR");
```

Comment:
Dit heb je uit de uitwerkingen gekopieerd, maar die waren helaas niet helemaal up-to-date. Kijk nog eens bij de uitwerkingen om te zien hoe het nu werkt